src/apps/allbridge/api.ts

```diff
@@ -98,7 +99,41 @@ export async function getAllbridgeTokenInfo({
   ) as AllbridgeApiResponse
   if (!allbridgeTokensInfoResponse) {
     allbridgeTokensInfoResponse = await got
-      .get('https://core.api.allbridgecoreapi.net/token-info')
+      .get('https://core.api.allbridgecoreapi.net/token-info', {
+        hooks: {
+          beforeRequest: [
+            (options) => {
+              logger.debug('allbridge: request details', {
+                url: options.url.toString(),
+                headers: options.headers,
```

Comment:
we don't use any api tokens / authentication in the headers that could be logged here right?